libethereum/Executive.h

```diff
@@ -78,6 +78,7 @@ class StandardTrace
 	DebugOptions m_options;
 };
 
+enum class CreationContext: short;
```

Comment:
I think you don't need and shouldn't duplicate `: short` here in the forward declaration (this is another nice feature of `enum class`)